### ISSUE TITLE
Make ControlNet loader optional if no image or controlnet_conditioning_scale is 0 - it can save VRAM

### ISSUE THREAD

**Issue by hortom** [ZHO-ZHO-ZHO/ComfyUI-InstantID#69]
Hi,
Is it possible to make ContorlNet optional and save VRAM? So, InstantID would be able to run on 12GB GPUs.

If `controlnet_conditioning_scale` is 0 or no connected `pose_image_optional` then ControlNet looks unnecessary.

If the above is correct, please amend your solution.

Cheers

**Comment by cerarslan** [comment]
Actually, I can run with 3060 12gb but not always. It is not stable

**Comment by hortom** [comment]
Yes, same here with 4070. Let me add "stable" to my first comment.

**Comment by hortom** [comment]
> Actually, I can run with 3060 12gb but not always. It is not stable

Yes, same here with 4070. Let me add "stable" to my first comment.

**Comment by hortom** [comment]
PR is here: https://github.com/ZHO-ZHO-ZHO/ComfyUI-InstantID/pull/87

**Comment by cerarslan** [comment]
Vaov thank you so much mate 🙏
